src/pperm.c

```diff
@@ -6322,9 +6212,18 @@ Obj OnSetsPPerm(Obj set, Obj f)
                     *ptres++ = tmp;
                 }
             }
-            else { /* this case cannot occur since I think POW is not defined
-                      */
-                ErrorQuit("not yet implemented!", 0L, 0L);
+            else {
+                // This case does not work since PowIntPPerm2/4 only work for
+                // INTOBJ's (i.e. small integers). The analogous code in
```

Comment:
We call them "small integers" or sometimes "immediate integers" elsewhere, and I'd stick to that terminology here, w/o brining it up again.

Reply:
Ok.